### DISCUSSION TITLE
为什么啊

### DISCUSSION THREAD

**Discussion by jian-li-5205** [share121/inter-knot#2709] [其他]
![image](https://github.com/user-attachments/assets/374dafaa-6867-4345-9fc0-16bc61dc27fb)

**Comment by sky-R6** [comment]
你是故意的還是不小心的?

**Comment by cyberrobber** [comment]
你得填到街道才行

**Comment by jian-li-5205** [comment]
这么智能的吗

**Reply by homteam** [reply]
> 这么智能的吗

严重怀疑你在逗我们

**Reply by 14-BOOT** [reply]
有邮箱吗？没有先去注册一个，填邮箱地址就好了

**Comment by fa-e-tong-LKN** [comment]
不出意外的话你**人机验证**得卡个1小时以上

**Reply by weixingstar** [reply]
曾经我被这玩意真红温了都

**Reply by ysd1911** [reply]
挂个加速器就好了吧，当时我也是搞了半小时，后来发现一挂加速器就彳亍了，主要是网络差就会让人一直搞
